example/dubbo/go-client/cmd/client.go

```diff
@@ -36,6 +38,7 @@ import (
 	_ "dubbo.apache.org/dubbo-go/v3/registry/protocol"
 	_ "dubbo.apache.org/dubbo-go/v3/registry/zookeeper"
 	"go.uber.org/atomic"
+	_ "net/http/pprof"
```

Comment:
maybe, you should move this line to 1st block

Reply:
done